ios/FluentUI/Label/Label.swift

```diff
@@ -42,7 +42,7 @@ open class Label: UILabel {
             updateTextColor()
         }
     }
-    @objc open var style: TextStyle = .body {
+    @objc open var style: AliasTokens.TypographyTokens = .body1 {
```

Comment:
It's okay that this class isn't "fully" tokenized yet, right? It won't respond to theme changes, for example.

Reply:
that is correct.